### PR TITLE
bump MAVEN to 3.9.6

### PR DESCRIPTION
**Pull request by qmonmert**

### CODE REVIEW

File: src/main/resources/generator/buildtool/maven/pom.xml.mustache [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-    <maven.version>3.9.5</maven.version>
+    <maven.version>3.9.6</maven.version>
```
Comment (murdos) [inline src/main/resources/generator/buildtool/maven/pom.xml.mustache:21]: I don't see the reason to require users to always use the latest maven version.
Everybody does not use maven wrapper, and does not jump on latest release as soon as it's release.
So IMO we should be more flexible, and even require a 3.8.0 version.

Comment (qmonmert) [reply src/main/resources/generator/buildtool/maven/pom.xml.mustache:21]: thank you for your opinion

**Comment by qmonmert** [comment]
@pascalgrimaud @murdos so we close this?

**Comment by pascalgrimaud** [comment]
I agree with @murdos : we should not force the maven version here, as some users could use an installed version, instead of the wrapper

**Comment by qmonmert** [comment]
@pascalgrimaud ok I close

**Pull request closed without merging**